src/HubCollection.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Mercure Component project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Mercure;
+
+use Symfony\Component\Mercure\Exception\InvalidArgumentException;
+
+final class HubCollection
```

Comment:
not sure about the name of this "service" 🤔

Reply:
may I suggest `HubRegistry`?

Reply:
i think that's a better fit 👍 done